sdk/keyvault/Azure.ResourceManager.KeyVault/api/Azure.ResourceManager.KeyVault.net8.0.cs

```diff
@@ -18,7 +18,7 @@ protected DeletedKeyVaultCollection() { }
     }
     public partial class DeletedKeyVaultData : Azure.ResourceManager.Models.ResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.KeyVault.DeletedKeyVaultData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.KeyVault.DeletedKeyVaultData>
     {
-        internal DeletedKeyVaultData() { }
+        public DeletedKeyVaultData() { }
```

Comment:
this might be incorrect.
I think this model is mistakenly added the `input` usage.
We need to remove this usage.

Reply:
The usage flag would not work.
I use another way to fix this in directive.